.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-    - "lts/dubnium"
+    - "--lts"
```

Comment:
It seems like this might have been connected to the Verizon outage yesterday. Should work with both --lts, lts/dubnium, 12 and lts/* now.